platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/State.java

```diff
@@ -240,9 +243,6 @@ public String getInfoString(final int hashDepth) {
      */
     @Override
     public String toString() {
-        return new ToStringBuilder(this)
-                .append("platformState", getPlatformState())
-                .append("swirldState", getSwirldState())
-                .toString();
+        return new ToStringBuilder(this).append("swirldState", getSwirldState()).toString();
```

Comment:
As far as I can tell, `PlatformState` is still part of this class, right? Why was it removed from the `toString` method?

Reply:
For no good reason. I returned it.